packages/bot-engine/src/scratch/help-content/help-content.config.js

```diff
@@ -1,8 +1,13 @@
-import constant from '../../constants';
+export const help_content_type   = {
```

Comment:
Can we make this plural?

```suggestion
export const help_content_types   = {
```